src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -489,7 +489,7 @@ private TypeAndRefKind GetTypeAndRefKind(ConsList<FieldSymbol> fieldsBeingBound)
                 binder = binder.WithAdditionalFlagsAndContainingMemberOrLambda(BinderFlags.SuppressConstraintChecks, this);
                 if (!ContainingType.IsScriptClass)
                 {
-                    var typeOnly = typeSyntax.SkipScoped(out _).SkipRef(out refKind);
+                    var typeOnly = typeSyntax.SkipScoped(out _).SkipRef(diagnostics, out refKind);
```

Comment:
>diagnostics [](http://example.com/codeflow?start=72&length=11)

Do we need this check since we'll check for `MessageID.IDS_FeatureRefFields` below?

Reply:
in this case, looks like it can be removed.